src/cli/self_update/windows.rs

```diff
@@ -285,6 +285,36 @@ pub fn do_remove_from_path() -> Result<()> {
     _apply_new_path(new_path)
 }
 
+/// Encodes a utf-8 string as a null-terminated UCS-2 string in bytes
```

Comment:
I think this comment is wrong though yes? the input isn't a utf8 string ;)

Reply:
Hope it is good now, not really sure of how to describe the operation